libs/designsystem/src/lib/components/page/page.component.scss

```diff
@@ -23,6 +27,41 @@ ion-toolbar {
   --padding-top: 0;
   --padding-bottom: 0;
 
+  // The toolbar height for iPhone's with a notched display.
+  $toolbar-height-notch: utils.size('xxl');
+
+  // The toolbar height for other displays.
+  $toolbar-height-default: utils.size('xxxl');
+
+  // An iPhone with a ios-safe-area-top exceeding 32px is considered to have notched display.
+  $notch-safe-area-top-trigger: utils.size('l');
```

Comment:
I read your argument from before, and I can see why it might not make sense to tie this to a variable. 
I think it would be good to revert to your initial solution with just a plain pixel value here.